org.eclipse.xtext.ide/src/org/eclipse/xtext/ide/editor/quickfix/DiagnosticResolution.java

```diff
@@ -65,11 +64,11 @@ public DiagnosticResolution(String label, Factory modificationContextFactory, IT
 		this.relevance = relevance;
 	}
 
-	public DiagnosticResolution(String label, Factory modificationContextFactory, IModification<EObject> modification) {
```

Comment:
these are both breaking changes, if someone uses this API. Shall I create additional methods to maintain backwards compatibility?

Reply:
maybe you can keep one,
and mark it with
`@Deprecated//(forRemoval=true,since="2.27") `

Reply:
Thanks, I did that (without commenting the properties of the annotation, since we require Java 11 I think it is not needed)

Reply:
we are still on java 8

Reply:
thx, fixed